ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml

```diff
@@ -2095,6 +2095,24 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-baremetalds-ipi-ovn-ipv4
+- as: baremetal-sno-agent-ipv4-static-connected-p1-f14
+  cluster: build05
+  cron: 30 21 * * 2
```

Comment:
Runs on every Tuesday, does not match the frequency f14.

Reply:
Fixed, thanks for review